### PR TITLE
add main.cpp

### PR DESCRIPTION
**Pull request by psychocoderHPC**
add `main.cpp` (includes `main.cu`)

### CODE REVIEW

File: src/libPMacc/examples/gameOfLife2D/main.cpp [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License and the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * and the GNU Lesser General Public License along with libPMacc.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "main.cpp"
```
Comment (ax3l) [inline src/libPMacc/examples/gameOfLife2D/main.cpp]: > add main.cpp (includes main.cu)

nope, it includes itself o.0

Comment (erikzenker) [inline src/libPMacc/examples/gameOfLife2D/main.cpp]: Well spotted

Comment (psychocoderHPC) [inline src/libPMacc/examples/gameOfLife2D/main.cpp]: fixed

**Comment by erikzenker** [comment]
Thank you! Ready to merge :-)

**Pull request merged**